integration-tests/scripts/install.sh

```diff
@@ -54,7 +56,17 @@ export BOMBASTIC_API_URL
 export OIDC_CLIENT_ID
 export OIDC_CLIENT_SECRET
 export OIDC_ISSUER_URL
-export SUPPORTED_CYCLONEDX_VERSION="1.4"
+## variables for Artifactory
+export ARTIFACTORY_URL
+export ARTIFACTORY_TOKEN
+export ARTIFACTORY_DOCKERCONFIGJSON
+export ARTIFACTORY_USERNAME
+export ARTIFACTORY_PASSWORD
+## variables for Nexus
+export NEXUS_URL
+export NEXUS_DOCKERCONFIGJSON
+export NEXUS_USERNAME
+export NEXUS_PASSWORD
```

Comment:
Too many env variable. Can we just have all these into a file and export the file content in this script